great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -1285,6 +1546,25 @@ def _register_metric_functions(cls):
                             metric_provider=_pandas_column_map_condition_value_counts,
                             metric_fn_type=MetricFunctionTypes.VALUE,
                         )
+                    elif domain_type == MetricDomainTypes.MULTICOLUMN:
+                        register_metric(
+                            metric_name=metric_name + ".unexpected_values",
+                            metric_domain_keys=metric_domain_keys,
+                            metric_value_keys=(*metric_value_keys, "result_format"),
+                            execution_engine=engine,
+                            metric_class=cls,
+                            metric_provider=_pandas_multicolumn_map_condition_values,
+                            metric_fn_type=MetricFunctionTypes.VALUE,
+                        )
+                        register_metric(
```

Comment:
I like the availability of this metric, but see caveat below.

Reply:
No caveat came, because it looks like we're not actually requesting this metric anywhere, right?

Reply:
@jcampbell Correct -- we will be requesting it in the upcoming PR.